packages/eslint-plugin/src/rules/index.ts

```diff
@@ -128,7 +128,7 @@ import unboundMethod from './unbound-method';
 import unifiedSignatures from './unified-signatures';
 import useUnknownInCatchCallbackVariable from './use-unknown-in-catch-callback-variable';
 
-export default {
```

Comment:
Why did you switch this from an `export default`?

Reply:
See [this comment](https://github.com/typescript-eslint/typescript-eslint/pull/10049#discussion_r1775961226).